scripts/rpm/fapolicy-analyzer.spec.j2

```diff
@@ -51,8 +51,6 @@ VERSION=%{app_version} %{python3} setup.py bdist_wheel
 %{python3} -m pip install --no-deps --no-input -v --target %{buildroot}%{python3_sitelib} dist/*.whl
 %{python3} -m pip install --no-deps --no-input -v --target %{buildroot}%{python3_sitelib} vendor/ReduxPY
 install bin/fapolicy-analyzer %{buildroot}%{_sbindir}/fapolicy-analyzer -D
-install share/sourceview/language-specs/fapolicyd-rules.lang %{buildroot}%{_datadir}/gtksourceview-3.0/language-specs/fapolicyd-rules.lang -D
-install share/sourceview/styles/fapolicyd.xml %{buildroot}%{_datadir}/gtksourceview-3.0/styles/fapolicyd.xml -D
```

Comment:
You'll need to remove the sourceview entry under `%files` below.

Reply:
Just found that. Thx.